yb-voyager/cmd/endMigrationCommand.go

```diff
@@ -429,18 +430,18 @@ func saveDataImportReport(msr *metadb.MigrationStatusRecord) {
 		return
 	}
 
-	importDataReportFilePath := filepath.Join(backupDir, "reports", "import_data_report.txt")
+	importDataReportFilePath := filepath.Join(backupDir, "reports", "import_data_status_report.json")
 	if utils.FileOrFolderExists(importDataReportFilePath) {
 		utils.PrintAndLog("import data report is already present at %q", importDataReportFilePath)
 	}
 
 	utils.PrintAndLog("saving data import report...")
-	strCmd := fmt.Sprintf("yb-voyager import data status --export-dir %s", exportDir)
+	strCmd := fmt.Sprintf("yb-voyager import data status --export-dir %s --output-format json", exportDir)
 	importDataStatusCmd := exec.Command("bash", "-c", strCmd)
-	saveCommandOutput(importDataStatusCmd, "import data status", importDataStatusMsg, importDataReportFilePath)
+	saveCommandGeneratedReport(importDataStatusCmd, "import data status", "import-data-status-report", importDataStatusMsg, importDataReportFilePath)
```

Comment:
wouldn't this also put some garbage info like
```
Using export-dir:....
```
whereas we would want to just backup the json report